CRM/Utils/Money.php

```diff
@@ -62,9 +62,12 @@ public static function format($amount, $currency = NULL, $format = NULL, $onlyNu
     }
 
     if ($onlyNumber) {
+      if ($valueFormat === '!%i') {
+        $amount = self::formatNumericByNumberFormatter($amount);
+      }
       // money_format() exists only in certain PHP install (CRM-650)
       if (is_numeric($amount) and function_exists('money_format')) {
-        $amount = money_format($valueFormat, $amount);
+        $amount = self::formatNumericByFormat($amount, $valueFormat);
```

Comment:
Just some standardisation / re-use of functions

Reply:
Should `if [..] function_exists('money_format')`   also be removed?